### PR TITLE
some restructuring

### PR DESCRIPTION
**Pull request by yevgenypats**
Mainly trying to put the same structure for each command we have:

```
--> CLI
    ---> command
        --> Overview
        --> Configuration
```

Also updating key features & use-cases as they were pretty old and not cristal clear.

### CODE REVIEW

File: docs/cli/drift/configuration.md [hunk 1]
```diff
@@ -99,9 +99,17 @@ provider "aws" {
...
 
 ```hcl
-provider "*" {
-  resource "*" {
-    deep = true // Deep mode for every resource and provider. (Essentially the same as --deep)
+modules {
+  // drift configuration block
+  drift "drift-example" {
+    // state block defines from where to access the state
+    terraform {
```
Comment (disq) [inline docs/cli/drift/configuration.md:106]: This is wrong. `terraform {` is its own block and it defines where to access tfstate files.

File: docs/cli/drift/overview.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
-# Drifting
+# Overview
 
-"Drifting" is a term used to detect infrastructure drifts, when the infra is managed by an IaC (infrastructure as code) provider such as [Terraform](https://www.terraform.io/).
+CloudQuery CLI can be used to detect and alert on IaC (Infrastructure-as-code, Terraform, pulumi, etc...) drift both locally and in the CI.
+
+**Infrastructure Drift** - When what is defined in your IaC is different from what is provisioned in your Cloud. This is a very command scenario and can happen due human error, manual changes and various other reasons. Checkout more details in our [blog](https://cloudquery.io/blog/announcing-drift-detection)
```
Comment (disq) [inline docs/cli/drift/overview.md:5]: suggestion: Prepend with `> ` ("quoted") so it's shown in a box

File: docs/cli/drift/configuration.md (hunk 1 above)
Comment (yevgenypats) [reply docs/cli/drift/configuration.md:106]: I basically just move this file https://github.com/cloudquery/docs/pull/36/files#diff-d74864886c0044889d03d46f938f63f150461311265acf909b262755ed547475R16 to `cli/drift/configuration`.

is it incorrect their as well ?

Comment (yevgenypats) [reply docs/cli/drift/configuration.md:106]: Ah sorry, I see it was updated. I'll rebase.

File: docs/cli/drift/overview.md (hunk 2 above)
Comment (yevgenypats) [reply docs/cli/drift/overview.md]: Done.

**Comment by yevgenypats** [comment]
Ready for review.

There is still more work to do in various areas of the documentation but I think it's best to do that in steps and different PRs (https://github.com/cloudquery/docs/issues/38 and some other places where we have a bit outdated documentation).

File: docs/cli/fetch/configuration.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Configuration
+
+CloudQuery just like [terraform](https://www.terraform.io/docs/language/providers/index.html) relies on "providers" to extract,transform and load cloud assets configuration from Cloud Providers, SaaS Providers and other APIs into PostgreSQL.
+
+This section is very similar to terraform providers section due to similar design of CloudQuery providers, but their purpose and implementation is completly different:
```
Comment (disq) [inline docs/cli/fetch/configuration.md:5]:
```suggestion
This section is very similar to terraform providers section due to similar design of CloudQuery providers, but their purpose and implementation is completely different:
```

File: docs/cli/fetch/configuration.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+CloudQuery providers are read only providers which extracts,transform and loads cloud assets configuration while Terraform providiers interacts with the cloud to provision assets.
```
Comment (disq) [inline docs/cli/fetch/configuration.md:7]:
```suggestion
CloudQuery providers are read only providers which extracts, transforms and loads cloud assets configuration while Terraform providers interact with the cloud to provision assets.
```

**Review by disq — CHANGES_REQUESTED** [review]

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**